packages/core/src/hooks/use_item_callbacks.js

```diff
@@ -0,0 +1,25 @@
+const useItemCallbacks = ({
```

Comment:
This is bit a custom hook, not sure we should use the same useCustomHook pattern.
Also, you are passing your own functions so no reason for it to return an array and not an object of these functions.

Reply:
You can add the state into this method and in that case this would make sense as a custom hook.